src/utilities/strategy-helpers/deployment-helper.ts

```diff
@@ -55,8 +55,9 @@ export async function deploy(kubectl: Kubectl, manifestFilePaths: string[], depl
         core.debug("Unable to parse pods; Error: " + e);
     }
 
-    await annotateAndLabelResources(deployedManifestFiles, kubectl, resourceTypes, allPods);
-    await AzureTraceabilityHelper.addTraceability(kubectl);
+    const deploymentConfig = await getDeploymentConfig();
+    await annotateAndLabelResources(deployedManifestFiles, kubectl, resourceTypes, allPods, deploymentConfig);
+    await AzureTraceabilityHelper.addTraceability(kubectl, deploymentConfig);

```

Comment:
Make this change in other references of `addTraceability` (I believe `promote` and `reject`)